pennylane/qchem/basis_set.py

```diff
@@ -122,7 +128,7 @@ def atom_basis_data(name, atom):
     return params
 
 
-def mol_basis_data(name, symbols):
+def mol_basis_data(name, symbols, load_data=False):
```

Comment:
I think it may be worthwhile to change the docstrings for functions that depend on basis sets, like for building molecule objects or Hamiltonians, to show that users can now use bse for loading any basis set. Not sure, but worth considering

Reply:
Added the argument to `molecular_hamiltonian`.